src/components/Header-Termsofuse/HeaderTermsofuse.jsx

```diff
@@ -1,40 +1,41 @@
 import React from 'react';
-import termsofuseLogo from '../../images/termsofuse-logo.png';
-import tan from '../../images/tan.png';
+import termsofuseLogo from '../../images/logo-for-gallery.png';
 import favorite from '../../images/input-icon-first.svg';
 import arrow from '../../images/input-second-icon.svg';
+import tan from '../../images/tann.png';
 
 const HeaderTermsofuse = () => {
   return (
     <div className='App-terms-of-use'>
       <div className='main-terms'>
```

Comment:
This input in header is not shown well, please fix this
![sinisa header](https://user-images.githubusercontent.com/85338076/213993674-9bc8e269-2da3-4584-a312-fba8be369b28.png)